cmd/registry/patch/apply.go

```diff
@@ -44,49 +56,53 @@ func Apply(ctx context.Context, client connection.RegistryClient, path, parent s
 			if err != nil {
 				return err
 			}
-			header, items, err := readHeaderWithItems(bytes)
+			return addPatches(client, patches, bytes, fileName, parent)
+		})
+	if err != nil {
+		return err
+	}
+	return patches.run(ctx, jobs)
+}
+
+func addPatches(client connection.RegistryClient, patches *patchGroup, bytes []byte, fileName, parent string) error {
```

Comment:
I think the role of this new function would be clearer if it was a method of patchGroup. Could you make that change and move this below the patchGroup struct declaration?